com.unity.cinemachine/Editor/Utility/CmPipelineComponentInspectorUtility.cs

```diff
@@ -0,0 +1,178 @@
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Cinemachine.Editor
+{
+    /// <summary>
+    /// Helpers for drawing CmComponentBase or CmExtension inspectors.
+    /// GML todo: make a common base class for CmComponentBase and CmExtension.
+    /// </summary>
+    class CmPipelineComponentInspectorUtility
+    {
+        UnityEditor.Editor m_Editor;
+
+        UnityEngine.Object[] Targets => m_Editor.targets;
```

Comment:
`UnityEngine.Object` could be replaced with `Object`. I don't mind it, because sometimes it's helpful to know.